src/Model.jl

```diff
@@ -168,9 +175,19 @@ function Model(lattice::AbstractMatrix{<:Quantity}, args...; kwargs...)
     Model(austrip.(lattice), args...; kwargs...)
 end
 
-normalize_magnetic_moment(::Nothing)  = Vec3{Float64}(zeros(3))
-normalize_magnetic_moment(mm::Number) = Vec3{Float64}(0, 0, mm)
-normalize_magnetic_moment(mm::AbstractVector) = Vec3{Float64}(mm)
+"""
+    Model(system::AbstractSystem; kwargs...)
+
+AtomsBase-compatible Model constructor. Sets structural information (`atoms`, `positions`,
+`lattice`, `n_electrons` etc.) from the passed `system`.
+"""
+Model(system::AbstractSystem; kwargs...) = call_with_system(Model, system; kwargs...)
```

Comment:
How about putting this and the other from standard_model into atomsbase? Generally it'd be nice to separate them to the max